quickstarts/setting-up-a-deployment.md

```diff
@@ -0,0 +1,30 @@
+## Setting Up A Deployment
+
+Before starting, make sure you have completed [setting up your environment](./quickstarts/getting-started.md)
+and [cleaening up your environment](./quickstarts/cleaning-up-an-environment.md).
+
+### Deploying profiles
+
+For an introduction to deploying profiles, see [Working with profiles](https://github.com/broadinstitute/dsp-k8s-deploy/blob/master/using-existing-profiles-quickstart.md#working-with-profiles) 
+
+### Deploy the shared profiles
+
+Before you can deploy your service(s), you will need to deploy the shared
+profiles that they need. The recommended order for deploying the shared profiles is:
+
+1. `network`
+2. `firewalls`
+3. `ssl`
+4. `populate-vault`
+
+These are the only shared profiles you should need for any service so far.
+
+### Deploy the service profiles
+
+For each service you want to deploy, there will likely be three profiles:
+
+1. `<service>-sa`: The service account and iam requirements. Deploy this first.
```

Comment:
Add that this must be done with the -a flag and the steps required to make that work